doc/Gym.md

```diff
@@ -1,13 +0,0 @@
-# OpenAI Gym wrappers
```

Comment:
Do we want to remove it if we keep the original gym wrapper?

Reply:
Good point, restored it